.github/workflows/build.yml

```diff
@@ -296,11 +292,70 @@ jobs:
         uses: codecov/codecov-action@eaaf4bedf32dbdc6b720b63067d99c4d77d6047d # v3.1.4
         if: always() && env.coverage == 'true'
 
+  pack:
+    needs: setup
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+
+      - name: Setup Node.js
+        uses: ./.github/actions/setup-node
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          os: ${{ runner.os }}
+
+      - name: Build
+        run: yarn -s build
+
+      - name: Pack
+        run: yarn -s test-e2e:pack
+
+      - name: Upload
+        uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
+        with:
+          name: renovate-package
+          path: renovate-v0.0.0-semantic-release.tgz
+
+  test-e2e:
+    needs: [setup, pack]
+    runs-on: ${{ matrix.os }}
```

Comment:
🤔 do we really need to test this on all platforms?

@rarkins WDYT?

Reply:
Okay, then removing other platforms. But I think end-to-end shouldn't use any caching, otherwise ends may not meet. So, I'll disable both `node_modules` and `yarn` global cache.